init-tools.cmd

```diff
@@ -26,8 +26,6 @@ if exist "%BUILD_TOOLS_SEMAPHORE%" (
   goto :EOF
 )
 
-if exist "%TOOLRUNTIME_DIR%" rmdir /S /Q "%TOOLRUNTIME_DIR%"
```

Comment:
What is the circumstance that leads to the state implied here... i.e. lack the BUILD_TOOLS_SEMAPHORE file, but a Tools directory we don't want to clean up.  

Lacking something you're working around, I preferred the old behavior since if that semaphore was not written that meant something bad happened between the creation of the tools directory and the finishing initialization of it.